generators/client/files-angular.js

```diff
@@ -111,6 +110,13 @@ const files = {
             templates: [
                 'blocks/interceptor/_auth.interceptor.ts'
             ]
+        },
+        {
+            condition: generator => !generator.skipServer,
```

Comment:
So what do you want me to do ? With these changes the app compiles with no error and run as expected